win32ss/user/user32/misc/exticon.c

```diff
@@ -616,6 +616,15 @@ static UINT ICO_ExtractIconExW(
             goto end;
         }
 
+#ifdef __REACTOS__
+        /* CORE-15879 fix by Thomas Faber */
```

Comment:
```suggestion
        /* Check for boundary limit (and overflow) */
```
and in the commit log, add:
```
 Co-authored-by: Thomas Faber <thomas.faber@reactos.org>
```

Reply:
Changed Line #620. Sorry, I do not know how to edit the commit log. Can you do this? Or tell me the git commands? Thanks.

Reply:
With the command-line, first do `git log` in order to find where the first commit of your PR starts (it should be `b84590a65f8b984ed31dbe7511a67197dd7ebf88`).
Then, do `git rebase -i b84590a65f8b984ed31dbe7511a67197dd7ebf88^` , which will open up a text editor (by default, vim, but you can configure another one).
Keep the first line (corresponding to that commit), and for the other (two) commits, replace the word `push` by `squash` (or just `s`), save and exit the editor.
Git will pick the changes, attempt to squash all commits together and will re-open the text editor for you to edit the commit log.
You will be able to give a proper commit title, and add any other modifications in the commit log (including adding that Co-authored-by line, that should be the very last one in the log, and followed by a newline.)

Reply:
@HBelusca, Thank you very much for your detailed description. I am using Windows 7 and "Git Bash Here" to get to a cmd prompt. When I entered you first 'git log' command, there were many lines of text that filled the screen. When I hit <enter> the next line appeared. I finally figured out that I had to hit "q" to quit and this exited the line-by-line display and got me back to the command prompt. Next I entered your next line 'git regase -i b8459..." and this opened notepad with the attached file. This file did not have any lines beginning with 'push', so I figured that I was stumped for now.
I have attached the file here for reference. Maybe you have another suggestion? Thanks.
[git-rebase-todo.txt](https://github.com/reactos/reactos/files/10835743/git-rebase-todo.txt)

Reply:
Please check now. I think that I succeeded using a combination of Tortoise git and 'Git bash here'. I selected the three commits in the log view of Tortoise Git and right-clicked and selected 'Combine to one commit'. This seemed to fix my local copy, so I used the 'Git bash here' prompt to do the ```git push --force-with-lease``` and this seemed to work. Thanks.

Reply:
I forgot to add the 'CORE-15879' reference. Done now. Thanks.